src/teststat/StatisticSum.cpp

```diff
@@ -0,0 +1,75 @@
+#include <StatisticSum.h>
+#include <ContainerTools.hpp>
+
+StatisticSum::StatisticSum(TestStatistic& s1_, TestStatistic& s2_){
+    AddStat(s1_);
+    AddStat(s2_);
+}
+
+double
+StatisticSum::Evaluate(){
+    double sum = 0;
+    for(size_t i = 0; i < fStats.size(); i++)
+        sum += fStats.at(i) -> Evaluate();
```

Comment:
This will become z = log(x) + log(y), in the case of BinnedNLLHs won't it, so z isn't a likelihood. Which is fine but I'm just checking.